### PR TITLE
[CORE] Add no-decimal TPC-DS queries

### PR DESCRIPTION
**Pull request by rui-mo**
## What changes were proposed in this pull request?

Add no-decimal TPC-DS queries, in which date is kept.

### CODE REVIEW

File: gluten-core/src/test/resources/tpcds-queries/tpcds.queries.no-decimal/q19.sql
Comment (zhouyuan) [inline gluten-core/src/test/resources/tpcds-queries/tpcds.queries.no-decimal/q19.sql:1]: maybe keep the old STRING DATE queries? those maybe still helpful

Comment (rui-mo) [reply gluten-core/src/test/resources/tpcds-queries/tpcds.queries.no-decimal/q19.sql:1]: I found the queries in repo were original ones. Do we need to add STRING_DATE ones to the repo also?

**Comment by jinchengchenghh** [comment]
I think we don't need to add them, just TPCDS original query is enough, we will support decimal soon

**Comment by FelixYBW** [comment]
> e don't need to add them, just TPCDS original query is enough, we will support decima

Let's still use double till decimal is supported.

**Review by FelixYBW — APPROVED** [review]

**Comment by rui-mo** [comment]
> Non-decimal is the one convert decimal to double. original is the official TPCDS?

Hi @FelixYBW, with this patch, we will have below three kinds of TPC-DS queries in Gluten repo.

- The double and string date queries, used in CI test: https://github.com/oap-project/gluten/tree/main/backends-velox/workload/tpcds/tpcds.queries.updated.
- Non-decimal queries, with double and date.
- Original queries, the official TPC-DS.

We also have two kinds TPC-H queries in current repo.

- The double and string date ones, also used in CI test.
- The offical TPC-H queries.

**Comment by FelixYBW** [comment]
> * The double and string date queries, used in CI test: [`main`/backends-velox/workload/tpcds/tpcds.queries.updated](https://github.com/oap-project/gluten/tree/main/backends-velox/workload/tpcds/tpcds.queries.updated?rgh-link-date=2023-03-22T01%3A42%3A40Z).
> * Non-decimal queries, with double and date.
> * Original queries, the official TPC-DS.
> 
> We also have two kinds of TPC-H queries in current repo.
> 
> * The double and string date ones, also used in CI test.
> * The offical TPC-H queries.

Thank you, @rui-mo Let's document this somewher as well.

**Review by FelixYBW — DISMISSED** [review]

**Review by FelixYBW — APPROVED** [review]

**Review by PHILO-HE — APPROVED** [review]

**Pull request merged**